collaboration.html

```diff
@@ -0,0 +1,171 @@
+						<li class="advisory">Utilize the information gained from the student’s IEP in the development of the IPE.</li>
+						<li class="advisory">Individualize planning for each student.</li>
+						<li class="advisory">Actively involve the student in their IEP and IPE meetings.</li>
+					</ul>
+					<p class="collapse"><a href="#" class="readless">Read less &gt;</a><hr></p>
+				</div>
+
+				<div class="container">
+					<h2>Potential roadblocks/suggested approaches<span class="expand"><a href="#"> +</a></span></h2>
+					<p class="advisory content">Team members may not understand the roles of adult service agencies and how they connect to transition planning for students.<br>Suggested Approaches:</p>
+					<ul class="advisory content">
+						<li class="advisory">Develop a flow of services outlining the activities, services, and connections needed to take place each year of high school in order for students to reach their desired employment outcome. .</li>
+						<li class="advisory">Outline the roles and responsibilities of each partner.</li>
+						<li class="advisory">Offer cross-training opportunities for partners.</li>
+						<li class="advisory">Use data from various partners to determine activities and services at the system and individual level.</li>
+					</ul>
+					<p class="advisory content">Adult service agencies and schools have difficulty engaging family members in the transition planning process.<br>Suggetsed Approaches:</p>
+					<ul class="advisory content">
+						<li class="advisory">Utilize the information gained from the student’s IEP in the development of the IPE.</li>
+						<li class="advisory">Individualize planning for each student.</li>
+						<li class="advisory">Actively involve the student in their IEP and IPE meetings.</li>
+					</ul>
+					<p class="collapse"><a href="#" class="readless">Read less &gt;</a><hr></p>
+				</div>
+
+					<h2>Tools</h2>
+					<h2>Thinking points</h2>
+				
+		</section>
+
+
+
+		<footer>
+			
+			<section class="clearfix">
+				<aside class="meet">
+					<p class="navfooter">Funded by:</p>
+					<a href="http://www.acl.gov/programs/nidilrr/"><img src="images/NIDILRR-logo-color.jpg" alt="NIDILRR logo" class="meet_img"></a>
+				</aside>
+				<aside class="meet">
+					<p class="navfooter">Administered by:</p>
+					<a href="www.transcen.org"><img src="images/TransCenLogoTrans.png" alt="TransCen logo" class="meet_img"></a>
+				</aside>
+				<aside class="meet">
+					<p class="navfooter">In partnership with:</p>
+					<a href="http://www.vcu.edu/"><img src="images/VCU_3CSeal.png" alt="Virginia Commonwealth University log" class="meet_img"></a>
+				</aside>
+			</section>
+
+			<div class="sitemap"><a href="index.html">Home</a> | <a href="aboutus.html">About Us</a> | <a href="toolkit.html">Toolkit</a> | <a href="contactus.html">Contact</a></div>
+
+			<div>
+				<p class="funding">Funded by Department of Education - PR/Award #H133A100007 CFDA #84.133A</p>
```

Comment:
I believe you should use the html entity for `#` symbol. Try `&num;` instead of `#`